### DISCUSSION TITLE
How to make Textfield unscrollable

### DISCUSSION THREAD

**Discussion by luiz00martins** [Pyroxenium/Basalt#32] [Q&A] [answered]
I have a layout, with some lines of Textfields and Dropdowns. There are enough lines such that you may need to scroll the frame.

Scrolling with the mouse on the main Frame works, as well as scrolling with the mouse on the Dropdown. However, scrolling on the Textfield doesn't, as it tries to scroll the Textfield, instead of the Frame behind it:

https://user-images.githubusercontent.com/43142209/196072577-31d91f00-1fb8-4511-99b4-9882a4445e12.mp4

Is there a way to make the Textfields unscrollable, such that the Frame is scrolled instead?

**Comment by NoryiE** [comment, marked as answer]
You could add :onScroll(function() return false end) to your textfield. Maybe thats what you're looking for.
